src/games/seabattle/game.test.tsx

```diff
@@ -35,6 +35,16 @@ describe('Seabattle', () => {
     }).toThrow();
   });
 
+  it('should only allow unique IDs', () => {
+    const client = Client({
+      game: SeabattleGame,
+    });
+    client.moves.setShips(VALID_SETUP_FIRST_PLAYER);
```

Comment:
Don't you need to update this VALID_SETUP_FIRST_PLAYER with random IDs?

Reply:
Yes, I forgot to commit the mock.ts -- fixed